src/chainparams.cpp

```diff
@@ -416,6 +422,9 @@ class CRegTestParams : public CChainParams {
         consensus.vUpgrades[Consensus::UPGRADE_SAPLING].nProtocolVersion = 170006;
         consensus.vUpgrades[Consensus::UPGRADE_SAPLING].nActivationHeight =
             Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT;
+        consensus.vUpgrades[Consensus::UPGRADE_BLOSSOM].nProtocolVersion = 170008;
```

Comment:
Perhaps we should leave a gap in case there is another P2P protocol upgrade before Blossom activation on testnet? Similarly, we could leave a gap between testnet and mainnet Blossom.

Reply:
We can always bump this later, before an activation height is set. The only place we use `nProtocolVersion` where we don't know what upgrade a peer is aware of, is during the handshake, and there we only use it to pre-emptively drop obsolete peers after an upgrade. If we bump this later, then anyone running a pre-bump version (which will have no activation height) will be correctly dropped as not supporting the upgrade.